src/Symfony/Component/Config/Tests/Definition/Builder/ArrayNodeDefinitionTest.php

```diff
@@ -357,6 +359,85 @@ public function testCannotBeEmptyOnConcreteNode()
         $node->getNode()->finalize([]);
     }
 
+    public function testFindShouldThrowExceptionIfNodeDoesNotExistInRootNode()
+    {
+        $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage('Node with name "child" does not exist in the current node "root"!');
+
+        $rootNode = new ArrayNodeDefinition('root');
+        $rootNode
+            ->children()
+                ->arrayNode('social_media_channels')->end()
+            ->end()
+        ;
+
+        $rootNode->find('child');
+    }
+
+    public function testFindShouldHandleComplexConfigurationProbably()
```

Comment:
you mean `Properly`?

Reply:
Your are probably right 🙈 Will fix it. Thanks @ro0NL 👍